components/Navbar.jsx

```
@@ -92,27 +92,29 @@ const Navbar = () => {
+              >
+                Sign Up
+              </Button>
+            </Link>
+          ))
+        }
       </div>
 
       {/* mobile nav menu */}
       <div className="lg:hidden flex flex-row gap-8 items-center transition-all">
-        {session.data && <NotificationBar />}
+        {(session.status = "authenticated" && <NotificationBar />)}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)